### PR TITLE
Added 'delay' parameter default and maximum values

### PR DESCRIPTION
**Pull request by vasilev**
Added 'delay' parameter default and maximum values for worker tasks and v3 messages.

### CODE REVIEW

**Comment by vasilev** [comment]
@treeder , could you please review ?

File: mq/3/reference/api/index.md [hunk 1]
```diff
@@ -358,6 +358,18 @@ Response:
...
+
+Multiple messages may be added in a single request, provided that the messages should all be added to the same queue. Each message object should contain the following keys:
+
+##### Required
+
+* **body**: The message data
+
+##### Optional
+
+* **delay**: The item will not be available on the queue until this many seconds have passed. Default is 0 seconds. Maximum is 604,800 seconds (7 days).
```
Comment (treeder) [inline mq/3/reference/api/index.md:371]: Looks good, but we should add all the available parameters here while we're at it. ping @convolvatron @rdallman

Comment (rdallman) [inline mq/3/reference/api/index.md:371]: just `delay` and `push_headers` which iirc are documented in `Push` section

**Comment by vasilev** [comment]
Thank you all !

**Pull request merged**